app/models/vm_or_template.rb

```diff
@@ -894,7 +887,7 @@ def reconnect_events
 
   def set_genealogy_parent(parent)
     with_relationship_type('genealogy') do
-      self.parent = parent
+      update!(:parent => parent)
```

Comment:
I feel like this method is now a near-duplicate of `genealogy_parent=`, and so should either be aliases of each other, or one calls the other.